### PR TITLE
Feature 3 surveys

### PR DESCRIPTION
**Pull request by kaeline**
https://github.com/Metatavu/oss-surveys-api-spec/issues/3
Inclues the ping endpoint from the other PR, should be merged after it

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+                type: array
+                items:
+                  $ref: "#/components/schemas/Survey"
+          headers:
+            X-Total-Count:
+              schema:
+                type: integer
+                format: int64
+              description: Total count of Surveys with given filters.
+        "400":
```
Comment (anttileppa) [inline swagger.yaml]: Error codes could be simplified by using default

File: swagger.yaml [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+        description: Payload
+        required: true
+      responses:
+        "200":
+          description: Created Survey
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Survey"
+        "400":
```
Comment (anttileppa) [inline swagger.yaml]: Error codes could be simplified by using default

File: swagger.yaml [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+            type: string
+            format: uuid
+      responses:
+        "200":
+          description: Survey
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Survey"
+        "400":
```
Comment (anttileppa) [inline swagger.yaml]: Error codes could be simplified by using default

File: swagger.yaml [hunk 4]
```diff
@@ -0,0 +1,314 @@
...
+            type: string
+            format: uuid
+      responses:
+        "200":
+          description: Updated Survey
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Survey"
+        "400":
```
Comment (anttileppa) [inline swagger.yaml]: Error codes could be simplified by using default

File: swagger.yaml [hunk 5]
```diff
@@ -0,0 +1,314 @@
...
+          required: true
+          in: path
+          description: Survey id
+          schema:
+            type: string
+            format: uuid
+      responses:
+        "204":
+          description: Success
+        "400":
```
Comment (anttileppa) [inline swagger.yaml]: Error codes could be simplified by using default

**Review by anttileppa — APPROVED** [review]

File: swagger.yaml (hunk 1 above)
Comment (kaeline) [reply swagger.yaml]: We probably should update the template project at some point, since it is copy paste from there

**Pull request merged**